ProfileAPI/ProfileAPI/Models/Profile.cs

```diff
@@ -0,0 +1,28 @@
+﻿using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace ProfileAPI.Models
+{
+    public class Profile
+    {
+        [BsonRepresentation(BsonType.ObjectId)]
+        public string Id { get; set; }
+
+        public string userPrincipalName { get; set; }
+
+        public string displayName { get; set; }
+
+        public string givenName { get; set; }
```

Comment:
Public properties should be upper case

Reply:
I wanted them to match those in the OfficeGraph but I can add json flag